internal/collector/configmap.go

```diff
@@ -47,7 +47,7 @@ var (
 		},
 		{
 			Name: "kube_configmap_created",
-			Type: metric.MetricTypeGauge,
+			Type: metric.Gauge,
```

Comment:
This actually fixes a golint issue. I guess this is concise and self-explanatory which is typical of idiomatic Go. 

`metric.MetricTypeGauge` is long and redundant.